app-frontend/src/app/components/map/annotateSidebarItem/annotateSidebarItem.component.js

```diff
@@ -0,0 +1,25 @@
+import annotateSidebarItemTpl from './annotateSidebarItem.html';
+
+const annotateSidebarItem = {
+    templateUrl: annotateSidebarItemTpl,
+    controller: 'AnnotateSidebarItemController',
+    bindings: {
+        mapId: '@',
```

Comment:
This list of bindings is definitely a bit unwieldy and a code smell - let's get a room and see if we can figure out a way to organize this better